xyzprinting_flexible.xml.fdm_material

```diff
@@ -0,0 +1,27 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material" xmlns:cura="http://www.ultimaker.com/cura">
+  <metadata>
+    <name>
+      <brand>XYZprinting</brand>
+      <material>Flexible</material>
```

Comment:
What actual material is this? TPU?

Reply:
It is TPEE,i will update this file,thank you :)